gers/examples/python/README.md

```diff
@@ -0,0 +1,70 @@
+# GERS Sidecar Match Examples
+
+## Context
+
+Consumers of geospatial data sets usually need to solve a complex and costly process of matching them.
+A data set that also has GERS IDs can be easily used to augment the Overture data set itself, or other data sets that also have GERS IDs via simple join by id.
+
+Because Overture data sets are modeled and produced with prioritizing for stability of its identifiers (GERS IDs) over time, and the cost of matching being offset to the owner of the data sets, the consumers of data sets with GERS IDs can conflate, evaluate and onboard such feeds much cheaper and faster.
+
+## Purpose
+
+Matching a geospatial data set with overture (or any other) data set is a common problem and many solutions exist for this, from generic to highly specialized for particular data types. 
+
+Depending on the match requirements, this can be achieved with a open source or commercial tools or services, with a few click or couple of lines of code or with large scale distributed system with complex match logic. 
+
+The purpose of these examples is not to provide a solution to fit all purposes well, or to be particularly good on any data scenario.
+
+Main purpose is to provide a few examples as options to start exploring your feed's compatibility with overture data set and to find GERS IDs that correspond to your features.
+Once you evaluate the results for sample of your feed, you can decide if the overlap with overture and match quality is sufficient for your feed with some tweaking of the available parameters, or if not, you can use a different solution that fits better your scenario, or customize the existing example code for your needs.
+
+## Types of Match
+
+Below are a few examples of match scenarios by category:
+
+- **MatchGeneric**: find most similar feature(s) of same or similar type from overture. Input types examples:
+   - Building -> Building
+   - Place -> Place
+   - Curbside parking info (road-like or road-adjacent LineStrings) -> Road segments
+- **Nearest**: sometimes simply finding nearest overture feature of a particular type can be valuable. Input types examples:

```

Comment:
these were meant to list different flavors of match for which we would gradually add more examples, but I'll remove everything and keep it about GPS trace match for now to avoid confusion